### PR TITLE
__get() method in search.js now uses lodash _.get() method

### PR DESCRIPTION
**Pull request by wahaj-47**

### CODE REVIEW

**Comment by wahaj-47** [comment]
We can also use attributes as plurals but we'll have to make a change in the staticFiles.test.js

**Comment by atherdon** [comment]
@wahaj-47 ok, do it too

**Comment by atherdon** [comment]
can you rename the method in search? i mean if we have __get in helper - this function should have different name - so we'll avoid any problems

**Comment by atherdon** [comment]
and change the name of const. it's not good

**Comment by wahaj-47** [comment]
> and change the name of const. it's not good

Are you referring to the const result ?

File: src/projects/Search/search.js [hunk 1]
```diff
@@ -5,37 +5,58 @@ const {
...
+//     'course': courses,
+//     'cuisine': cuisines,
+//     'diet': diets,
+//     'holiday': holidays,
+//     'ingredient': ingredients1,
+//     'measurement': measurements
+//   }
+// }
+
+const object = {
```
Comment (atherdon) [inline src/projects/Search/search.js]: and change the name of const.

File: src/projects/Search/search.js [hunk 2]
```diff
@@ -192,7 +213,7 @@ const toOptAntD = (data) => {
 }
 
 module.exports = {
```
Comment (atherdon) [inline src/projects/Search/search.js]: Check the export line. You forgot apply that changes on export too

**Comment by atherdon** [comment]
perfect

**Pull request merged**